src/.nuget/packages.targets

```diff
@@ -0,0 +1,288 @@
+
+            <!-- Uniqify package name per machine-local DNX cache for development purposes -->
+            <ToolchainPackageName Condition="'$(ToolchainMilestone)'=='testing'">$(ToolchainPackageName).$(BuildUid)</ToolchainPackageName>
+            <RuntimeSdkPackageName Condition="'$(ToolchainMilestone)'=='testing'">$(RuntimeSdkPackageName).$(BuildUid)</RuntimeSdkPackageName>
+
+            <LibraryPrefix Condition="'$(OSEnvironment)'=='Unix'">$(LibraryPrefix)lib</LibraryPrefix>
+            <LibraryExt Condition="'$(OSEnvironment)'!='Unix'">lib</LibraryExt>
+            <LibraryExt Condition="'$(OSEnvironment)'=='Unix'">a</LibraryExt>
+        </PropertyGroup>
+        <ItemGroup>
+            <!-- IL.Compiler target files -->
+            <ILCompilerFiles Include="ilc.exe" />
+            <ILCompilerFiles Include="ILCompiler.Compiler.dll" />
+            <ILCompilerFiles Include="ILCompiler.DependencyAnalysisFramework.dll" />
+            <ILCompilerFiles Include="ILCompiler.TypeSystem.dll" />
+            <ILCompilerBinPlace Include="@(ILCompilerFiles)">
+                <Text><![CDATA[        <file src="../%(Identity)" target="runtimes/any/lib/dotnet/%(Identity)" /> ]]></Text>
+            </ILCompilerBinPlace>
+
+            <!-- IL.Compiler.SDK target files -->
+            <ILCompilerSdkFiles Include="Runtime" />
+            <ILCompilerSdkFiles Include="PortableRuntime" />
+            <ILCompilerSdkFiles Include="bootstrapper" />
+            <ILCompilerSdkFiles Include="bootstrappercpp" />
+            <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'=='Unix'" />
+
+            <ILCompilerSdkFilesManaged Include="System.Private.CoreLib" />
+            <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFiles)">
+                <Text><![CDATA[        <file src="../lib/$(LibraryPrefix)%(Identity).$(LibraryExt)" target="runtimes/$(NuPkgRid)/native/sdk/$(LibraryPrefix)%(Identity).$(LibraryExt)" /> ]]></Text>
+            </ILCompilerSdkBinPlace>
+            <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFilesManaged)">
+                <Text><![CDATA[        <file src="../%(Identity).dll" target="runtimes/$(NuPkgRid)/native/sdk/%(Identity).dll" /> ]]></Text>
+            </ILCompilerSdkBinPlace>
+
+            <!-- ILCompiler nuspec file -->
+            <NuSpecFile Include="$(ToolchainPackageName)">
+                <Stage>0</Stage>
+                <Version>$(ToolchainVersion)</Version>
+                <Title>Microsoft .NET Native Toolchain</Title>
+                <Description>Provides the toolchain to compile managed code to native.</Description>
+                <Files>@(ILCompilerBinPlace -> '%(Text)', '')</Files>
+                <!-- TODO: Obtain this from project.lock.json -->
+                <Dependencies><![CDATA[
+        <dependency id="Microsoft.DiaSymReader" version="1.0.6" />
```

Comment:
SDK package should depend upon the toolchain as it will contain artifacts tied to the toolchain.